types/stylelint/index.d.ts

```diff
@@ -168,6 +170,10 @@ declare namespace stylelint {
 		severity?: Severity;
 		url?: string;
 		rule?: string;
+		fix?: {
+			range: [number, number];
+			text: string;
+		};
```

Comment:
Since you matched [`editInfo`](https://eslint.org/docs/latest/integrate/nodejs-api#-editinfo-type)'s shape and you have it twice, you can add

```ts
type EditInfo = {
	/** pair of 0-based indices etc. */
	range: [sourceStartOffset: number, sourceEndOffset: number];
	/** bar */
	text: string;
};
```

Reply:
I've added the `EditInfo` type, should still be documented

Reply:
I've added some docs.